sonar-application/src/main/assembly/conf/sonar.properties

```diff
@@ -255,7 +259,7 @@
 #socksProxyHost=
 #socksProxyPort=
 
-# proxy authentication. The 2 following properties are used for HTTP and SOCKS proxies.
+# Proxy authentication (used for HTTP, HTTPS and SOCKS proxies)
 #http.proxyUser=
 #http.proxyPassword=
```

Comment:
By design, you can't have a user/password specific to https or http or sock.

Is it because you can only configure one type of proxy at a time?